lib/tasks/migrate/20240715013839_add_team_id_to_claim_description.rake

```diff
@@ -0,0 +1,26 @@
+namespace :check do
+  namespace :migrate do
+    task add_team_id_to_claim_description: :environment do |_t, args|
+      started = Time.now.to_i
+      slugs = args.extras
+      condition = {}
+      if slugs.blank?
+        last_team_id = Rails.cache.read('check:migrate:add_team_id_to_claim_description:team_id') || 0
+      else
+        last_team_id = 0
+        condition = { slug: slugs }
+      end
+      Team.where(condition).where('id > ?', last_team_id).find_each do |team|
+        puts "Processing team [#{team.slug}]"
+        team.project_medias.joins(:claim_description).find_in_batches(batch_size: 2500) do |pms|
+          print '.'
+          ids = pms.map(&:id)
+          ClaimDescription.where(project_media_id: ids).update_all(team_id: team.id)
+        end
```

Comment:
@melsawy you don't need the `ProjectMedia`'s at all here, right? So I suggest you do something like this: https://github.com/meedan/check-api/blob/epic/CV2-4441-articles/lib/tasks/migrate/20240713012502_set_imported_for_fact_checks.rake#L6-L15. It will save memory, since loading the full project media objects in memory here is not needed. Something like:

`ClaimDescription.joins(:project_media).where('project_medias.team_id' => team.id).update_all(team_id: team.id).limit(BATCH_SIZE)`.

Reply:
I think my query do the same I started with team and select only ProjectMedias that have Claims.